addons/l10n_zm_account/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,528 @@
+                            </record>
+                        </field>
+                    </record>
+                    <record id="zm_tax_report_line_34" model="account.report.line">
+                        <field name="name">34 - Purchases from unregistered suppliers</field>
+                        <field name="code">zm_vat_34</field>
+                        <field name="expression_ids">
+                            <record id="zm_tax_report_line_34_no_vat" model="account.report.expression">
+                                <field name="label">no_vat</field>
+                                <field name="engine">tax_tags</field>
+                                <field name="formula">zm_vat_34_no_vat</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="zm_tax_report_line_35" model="account.report.line">
+                        <field name="name">35 - Total purchases</field>
+                        <field name="code">zm_vat_35</field>
+                        <field name="expression_ids">
+                            <record id="zm_tax_report_line_35_no_vat" model="account.report.expression">
+                                <field name="label">no_vat</field>
+                                <field name="engine">aggregation</field>
+                                <field name="formula">zm_vat_30.no_vat + zm_vat_31.no_vat + zm_vat_32.no_vat + zm_vat_33.no_vat + zm_vat_34.no_vat</field>
+                            </record>
+                        </field>
+                    </record>
+                </field>
+            </record>
+
+            <record id="zm_tax_report_title_input_tax" model="account.report.line">
+                <field name="name">Calculation of Input Tax Allowed</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="zm_tax_report_line_37" model="account.report.line">
+                        <field name="name">37 - Input Tax directly attributable to Taxable Sales</field>
+                        <field name="code">zm_vat_37</field>
+                        <field name="expression_ids">
+                            <record id="zm_tax_report_line_37_vat" model="account.report.expression">
+                                <field name="label">vat</field>
+                                <field name="engine">external</field>
+                                <field name="formula">sum</field>
+                                <field name="subformula">editable</field>
+                            </record>
+                        </field>
+                    </record>
+                    <record id="zm_tax_report_line_38" model="account.report.line">
+                        <field name="name">38 - Input Tax directly attributable to Exempt Sales</field>
+                        <field name="code">zm_vat_38</field>
```

Comment:
@clbr-odoo Do not we mostly put these codes with capital letters?

Reply:
I don't think we have any reliable best practice, I've seen a bit of everything...